### ISSUE TITLE
Peer dependency notation appears to be incorrect

### ISSUE THREAD

**Issue by lodybo** [Maloric/tslint-jasmine-noSkipOrFocus#4]
While updating this package to its latest version, I noticed something after running `npm install`:

````bash
npm ERR! peer dep missing: tslint@4.5.1-5.2.0, required by tslint-jasmine-noSkipOrFocus@1.0.7
````

I have `tslint` currently at `5.3.0`, but after downgrading to `5.2.0` and even `5.0.0` the error message still appeared. So I dove into the `package.json` of this package and saw the version range:
````javascript
"peerDependencies": {
    "tslint": "4.5.1-5.2.0"
  }
````

I ran the version range through [NPM's semver tester](https://semver.npmjs.com/) and saw that it didn't match. NPM is treating `4.5.1-5.2.0` as a single version, instead of a range. This can be fixed by simply adding spaces between the delimiter: `4.5.1 - 5.2.0`.

I have a fork with the fix standing by, but was wondering if you would like to have it accept versions upwards of `5.2.0` as well (being that `5.3.2` is the latest version).

**Comment by Maloric** [comment]
That seems perfectly reasonable, as long as it still works with 5.3.2.  Sorry for not responding sooner.

Open up a pull request and I'll be happy to review and merge it.

**Comment by lodybo** [comment]
No problem, I'll test it out tomorrow and open a pull request.

**Comment by lodybo** [comment]
It took me a bit longer than I wanted to, but I've created a pull request (#5) with my changes. I tested it against `tslint@5.5.0` to make it still works and it does, so I updated the notation to accept it upwards to that version.

**Comment by Maloric** [comment]
Fixed by PR #5
